src/portable/nuvoton/nuc121/dcd_nuc121.c

```diff
@@ -368,13 +381,15 @@ void dcd_edpt_clear_stall(uint8_t rhport, uint8_t ep_addr)
   (void) rhport;
   USBD_EP_T *ep = ep_entry(ep_addr, false);
   ep->CFG |= USBD_CFG_CSTALL_Msk;
+  ep->CFG &=~USBD_CFG_DSQSYNC_Msk;
 }
 
 void dcd_int_handler(uint8_t rhport)
 {
   (void) rhport;
 
-  uint32_t status = USBD->INTSTS;
+  // Mask non-enabled irqs, ex. SOF
+  uint32_t status = USBD->INTSTS & (enabled_irqs | 0xffffff00);
```

Comment:
could you explain why should we OR  enabled_irqs with 0xffffff00 here

Reply:
`enabled_irqs` cover only DCD itself's irqs, 0xffffff00 allows EP's irqs to occur.

Reply:
in that case, we should use this instead

```C
uint32_t status = USBD->INTSTS &  USBD->INTEN
```

Reply:
In `USBD->INTEN` there is no EPEVTx
![image](https://user-images.githubusercontent.com/4375114/133054815-79591290-79fc-4640-8a37-3a0ff21f1048.png)
![image](https://user-images.githubusercontent.com/4375114/133054839-19201c3d-e765-4e33-8686-2dc7c27d40c1.png)

Reply:
you are right, the nuc inten isn't same mapping as intsts as other mcu. Though, the value `0xffffff00` isn't quite correct, 0xffff0000 would be more precise (to also take `wken` into account, innaken is never used).

Reply:
update: ah I see, 0xffffff00 since wken and innaken is not present on the INTSTS :facepalm: